jvm/src/common/Language/Java/Internal.hs

```diff
@@ -155,3 +160,20 @@ getStaticFieldAsJValue retsing cname fname = do
     SPrim "double" -> JDouble <$> getStaticDoubleField klass field
     SVoid -> fail "getStaticField cannot yield an object of type void"
     _ -> JObject <$> getStaticObjectField klass field
+
+mkVariadic :: TH.TypeQ -> (TH.TypeQ -> TH.TypeQ -> [TH.PatQ] -> TH.ExpQ -> TH.ExpQ -> TH.DecsQ) -> TH.DecsQ
```

Comment:
I don't know the normal documentation standards in this code, but I would want more documentation here.

Reply:
It's an internal function that's only part of a separate module due to the stage restriction. Haddock won't help document the arguments of the continuation, but we can do that with a regular comment.